sql-plugin/src/main/scala/com/nvidia/spark/rapids/Plugin.scala

```diff
@@ -111,6 +115,41 @@ object RapidsPluginUtils extends Logging {
     }
   }
 
+  private def detectMultipleJar(propName: String, jarName: String, propsName: Seq[String], 
```

Comment:
NIT: The parameter name is a little confusing.
```suggestion
  private def detectMultipleJar(propFileName: String, jarName: String, propNames: Seq[String], 
```

Reply:
done.